recipe/patches/0007-enable-meson-build.patch

```diff
@@ -0,0 +1,32 @@
+From a7218a2df4164b56ffddfb14edd322fe94ab179e Mon Sep 17 00:00:00 2001
+From: Charles Bousseau <cbousseau@anaconda.com>
+Date: Wed, 14 Jun 2023 11:31:27 -0400
+Subject: [PATCH] enable meson build
+See: https://github.com/numpy/numpy/blob/v1.25.0rc1/building_with_meson.md
+---
+ pyproject.toml | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/pyproject.toml b/pyproject.toml
+index 759b538fb..6df577847 100644
+--- a/pyproject.toml
++++ b/pyproject.toml
+@@ -4,13 +4,13 @@
+ #build-backend = "mesonpy"
```

Comment:
You are right. I updated the patch.
And I am adding meson 1.1.1 to default because meson.build requires >=1.1.0
(https://github.com/AnacondaRecipes/meson-feedstock/pull/16)